test/test_package_version.py

```diff
@@ -63,7 +63,7 @@ def test_update_versions(self):
 
     def test_update_changelog_unicode(self):
         """Test that updating the changelog does not throw an exception on unicode characters."""
-        with tempfile.NamedTemporaryFile() as temp_file:
+        with tempfile.NamedTemporaryFile(delete=False) as temp_file:
```

Comment:
With this chance the temporary file is not being deleted at all. That is not the desired behavior.

An alternative would be to create a temporary directory with the context manager instead - to ensure the directory is removed after the test as well as files in that directory can be read after being written.

Reply:
Thank for the feedback! I changed it to use context manager.